### PR TITLE
beta -> latest (4.50.7)

### PR DESCRIPTION
**Pull request by bwp91**
My main focus here was updating package dependencies for the backend.

The dependencies in `ui/package.json` I have updated where it didnt bring any build errors.

### CODE REVIEW

**Comment by bwp91** [comment]
is it just me or have these dependency updates make the UI insanely faster?!

**Comment by bwp91** [comment]
oh it built this time, let's try once more

**Comment by bwp91** [comment]
Before I turn this draft to a 'ready' PR, wanted to get a couple of approvals, I mean it is all totally working for me having installed the betas that were published...

**Comment by bwp91** [comment]
back to draft...

Screenshot 2023-09-30 at 22 44 34

**Review by donavanbecker — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -52,76 +52,76 @@
...
   "dependencies": {
-    "@fastify/helmet": "9.1.0",
-    "@fastify/multipart": "7.1.0",
-    "@fastify/static": "6.5.0",
-    "@fastify/swagger": "7.4.1",
+    "@fastify/helmet": "11.1.1",
+    "@fastify/multipart": "8.0.0",
+    "@fastify/static": "6.11.2",
+    "@fastify/swagger": "8.11.0",
     "@homebridge/node-pty-prebuilt-multiarch": "0.10.1-pre.6",
```
Comment (NorthernMan54) [inline package.json]: This not ready for release yet.  The NPM package does not contain any of the prebuilt modules

File: package.json [hunk 2]
```diff
@@ -28,8 +28,8 @@
     "hb-service": "./dist/bin/hb-service.js"
   },
   "engines": {
-    "node": ">=16.20.2",
-    "homebridge": ">=1.6.0"
+    "node": "^18.18.0 || ^20.8.0",
```
Comment (NorthernMan54) [inline package.json]: FYI - The Synology DSM package uses NodeJS 18.15 ( so this generate a warning )

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,16 @@
+## BETA
```
Comment (NorthernMan54) [inline CHANGELOG.md]: I don't think is a BETA any more

File: package.json (hunk 2 above)
Comment (bwp91) [reply package.json]: i change to simply `"node": "^18 || ^20",`

File: CHANGELOG.md (hunk 3 above)
Comment (bwp91) [reply CHANGELOG.md]: @donavanbecker please comment on the specifics, but i think this version in the changelog is updated _just_before merging into `latest` branch?

Comment (donavanbecker) [reply CHANGELOG.md]: Technically this is beta because we haven't release 4.50.7 and that is the next version.

Comment (donavanbecker) [reply CHANGELOG.md]: Maybe add the Verison number then beta after or test?

`4.50.-test.*`?

Or

`Beta 4.50.7`?

Comment (bwp91) [reply CHANGELOG.md]: @donavanbecker but we wouldn’t normally do this in a changelog file right?

Comment (donavanbecker) [reply CHANGELOG.md]: Normally do what? Create the change log? I would say yes we would create it while we push out betas to test.

File: package.json (hunk 1 above)
Comment (NorthernMan54) [reply package.json]: Package has been updated to v0.11.1

**Comment by bwp91** [comment]
Need to downgrade fastify again, type error

**Comment by bwp91** [comment]
Closing, replaced by https://github.com/homebridge/homebridge-config-ui-x/pull/1585

**Pull request closed without merging**